roles/pulp-webserver/templates/pulp-vhost.conf.j2

```diff
@@ -32,6 +32,6 @@ Define pulp-api {{ pulp_api_host }}:{{ pulp_api_port }}
   ProxyPass /pulp/content http://${pulp-content}/pulp/content
   ProxyPassReverse /pulp/content http://${pulp-content}/pulp/content
 
-  IncludeOptional pulp/*.conf
+  IncludeOptional pulp/*
```

Comment:
I tested this when I introduced it and it worker for me. I'd like to test this part to see what's causing it to not work in your env before we merge.